lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -245,6 +286,10 @@ def main():
     task_mgr = EcsTaskManager(module)
     results = dict(changed=False)
 
+    if module.params['launch_type']:
+        if not task_mgr.ecs_api_supports_requirescompatibilities():
```

Comment:
This could also use `module.botocore_at_least()`

Reply:
It looks like the changes in 7f8654d where only for AnsibleAWSModule and ecs_taskdefinition is using AnsibleModule https://github.com/ansible/ansible/blob/6ef2ffe310b725b4f445fc0d95fff9954df94739/lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py#L239

I saw this when running a test
```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'AnsibleModule' object has no attribute 'botocore_at_least'
```

Reply:
We should move ecs_taskdefinition to AnsibleAWSModule but we can do that in a separate PR :)